packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/examples/PeoplePicker.Types.Example.tsx

```diff
@@ -38,7 +38,8 @@ const suggestionProps: IBasePickerSuggestionsProps = {
   noResultsFoundText: 'No results found',
   loadingText: 'Loading',
   showRemoveButtons: true,
-  suggestionsAvailableAlertText: 'People Picker Suggestions available'
+  suggestionsAvailableAlertText: 'People Picker Suggestions available',
+  suggestionsContainerAriaLabel: 'Suggested contacts'
```

Comment:
How is this different than the `suggestionsAvailableAlertText` prop that should already be ready when the suggestions box appears?

Reply:
The alert text is text that's read to alert the user there are suggestions shown. The new prop is to describe the containing div.